tests/unit_tests/conftest.py

```diff
@@ -39,12 +39,13 @@ def graph():
     )
     # Non-function differences
     g["do_check"].nonfun_diffs.append(ComparisonGraph.SyntaxDiff(
-        "MACRO", "do_check",
+        "macro", "___MACRO", "do_check",
```

Comment:
I'm not very familiar with this code but why did the `name` change from `MACRO` to `___MACRO`? Was it incorrect before?

Reply:
The tests should probably work, but I think it was not correct - as `name` of SyntaxDifference is in Simpll used name of the last/differing macro - in this test, it was not the case.